pcsx2/x86/ix86-32/iR5900-32.cpp

```diff
@@ -1669,6 +1687,9 @@ static void __fastcall recRecompile( const u32 startpc )
 			doPlace0Patches();
 		g_patchesNeedRedo = 0;
 	}
+	
+	if (g_eeloadExec && HWADDR(startpc) == HWADDR(g_eeloadExec))
```

Comment:
Maybe this location could be used to check for arguments. If none are given, then this could skip the new hook altogether, right?

Reply:
That would be more logical, right?  I was originally going to place the check here instead of in R5900.cpp's eeloadHook2(), but AppConfig.h is not included in this source file and so g_Conf is not known.  I was hesitant to change any includes in the project.  What would you suggest?

Reply:
Ah yeah. I would have decided against including AppConfig.h as well.
If you could add a comment that explains it, that'd be great :)

@refractionpcsx2 Your opinion on this? :)

Reply:
Why is that so? You want to request a config and accordingly add appconfig to this file then... sounds totally reasonable to me.

Reply:
If I was someone external, proposing my additions, I would try to avoid spreading headers. This is just how I'd do it though, not a technical reason.
I'm fine with the code as is ;)